src/gui/styles_dialog.c

```diff
@@ -705,20 +705,24 @@ GtkWidget *dt_gui_style_content_dialog(char *name, const int imgid)
   GtkWidget *label = NULL;
 
   // name
-  snprintf(buf, sizeof(buf), "<b>%s</b>", g_markup_escape_text(name, -1));
+  gchar *esc_name = g_markup_escape_text(name, -1);
```

Comment:
Could to this in one go with `g_markup_printf_escaped` (without need for buf)

Reply:
Thanks for the pointer, will do.

Reply:
Done